src/utils/validators/validateCustomDateRange.test.ts

```diff
@@ -0,0 +1,30 @@
+import { validateCustomDateRange } from "./validateCustomDateRange"
+
+describe("validateCustomDateRange", () => {
+  it("should return a date range for a valid from and to dates", () => {
```

Comment:
`should return a date range for valid from and to dates`? `dates` is plural, so no `a`?